composer.json

```
@@ -17,9 +17,9 @@
     ],
     "require": {
         "php": "^7.4|^8.0",
-        "illuminate/contracts": "^8.37",
-        "illuminate/support": "^8.37",
-        "illuminate/http": "^8.37",
+        "illuminate/contracts": "^8.37|^9.0",
+        "illuminate/support": "^8.37|^9.0",
+        "illuminate/http": "^8.37|^9.0",
         "symfony/process": "^5.1.4"
```

We should also reduce the constraint on Symfony Process as Laravel 9 requires Symfony 6.*

```suggestion
        "symfony/process": "^5.1.4|^6.0"
```